exec/java-exec/src/main/resources/rest/index.ftl

```diff
@@ -269,9 +277,12 @@
                               }
                         });
       }
-      <#if model.shouldShowAdminInfo() || !model.isAuthEnabled() >
-          function shutdown(address,button) {
-              url = "http://"+address+":"+portNum+"/gracefulShutdown";
+       <#if model.shouldShowAdminInfo() || !model.isAuthEnabled()>
+          function shutdown(button) {
+              var protocol = location.protocol;
+              var host = location.host;
+              var requestPath = "/gracefulShutdown";
+              var url = protocol+host+requestPath;
```

Comment:
How about refactoring above lines into separate function `getRequestUrl(requestPath)` and call that from both the methods (`shutdown` & `fillQueryCount`)

```
function getRequestUrl(requestPath) {
    var protocol = location.protocol;
    var host = location.host;
    var url = protocol + host + requestPath;
    return url;
}
```